app/admin/footer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module ActiveAdmin
+  module Views
+    class Footer < Component
+      def build(_namespace)
+        super id: 'footer'
+
+        div do
+          render partial: 'components/footer'
+        end
```

Comment:
J'ai l'impression que cette div supplémentaire ne sert à rien. Le footer sur l'écran de login est construit différemment sans cette div supplémentaire et cela fonctionne de la même manière.
```suggestion
          render partial: 'components/footer'
```